src/main/kotlin/lotto/business/LottoBookingSystem.kt

```diff
@@ -1,15 +1,34 @@
 package lotto.business
 
+import lotto.view.LottoInputHandler
+
 class LottoBookingSystem(
     private val randomLottoPicker: RandomLottoPicker = RandomLottoPicker()
 ) {
-
     private fun generateSingleTicket(): LottoTicket {
-        val lottoNumbers = randomLottoPicker.pick()
+        val lottoNumbers = randomLottoPicker.pick(LOTTO_NUMBERS, LOTTO_NUMBER_COUNT)
         return LottoTicket(lottoNumbers)
     }
 
-    fun generateMultipleTickets(count: Int): List<LottoTicket> {
-        return (1..count).map { generateSingleTicket() }
+    fun generateMultipleTickets(purchasableCount: Int): List<LottoTicket> {
+        return (1..purchasableCount).map { generateSingleTicket() }
+    }
+
+    fun generateManualTickets(manualCount: Int, purchasableCount: Int): List<LottoTicket> {
+        require(purchasableCount >= manualCount) { "구매 가능한 로또 개수보다 많습니다." }
+        return LottoTicketGenerator.generateManualTickets(LottoInputHandler.inputManualNumbers(manualCount))
+    }
```

Comment:
`LottoBookingSystem`은 도메인 객체 즉 model의 영역이라고 생각되는데요. 🤔
내부적으로는 수동 로또를 직접 입력받고있어요. 👀
LottoBookingSystem에서 직접적으로 입력을 받지않고 외부에서 주입받도록 변경해보면 어떨까요? 

또 한가지 얘기해보고 싶은건 수동 로또를 생성하는 시점에 `require(purchasableCount >= manualCount) { "구매 가능한 로또 개수보다 많습니다." }`를 확인할 필요가 있을까요?  `player.addTickets(manualLottoTickets)`을 호출하는 순간 메서드 내부의 유효성 검증에 의해서 똑같은 검증이 수행될 것 같아요. 

LottoBookingSystem은 단순히 수동 로또 갯수만큼 예약( 구입?)하는 것에만 관심을 두면 될 것 같아요. 😃
수도 로또 구입 갯수와 입력한 수동 로또의 갯수가 일치하는지 여부 정도는 여기서 검증을 해야할 수도 있을 것 같네요. 🤔
```suggestion
    fun generateManualTickets(manualCount: Int, ): List<LottoTicket> {
        require (manualCount == lottoTickets.size) { 적절한 예외메시지 }
        return LottoTicketGenerator.generateManualTickets(lottoTickets)
    }
```